halo2-common/src/lib.rs

```diff
@@ -0,0 +1,280 @@
+// Based upon https://github.com/privacy-scaling-explorations/snark-verifier/blob/main/snark-verifier/examples/evm-verifier-with-accumulator.rs
+
+use halo2_proofs::halo2curves as halo2_curves;
+
+pub mod application {
+    use super::halo2_curves::bn256::Fr;
```

Comment:
Line 3 could be deleted. And line 6 could 'simply' be
```suggestion
    use halo2_proofs::halo2curves::bn256::Fr;
```
super reference only used on line 6, and halo2_curves labeling only used on line 6.

Reply:
Removed